ament_cmake_core/cmake/core/package_xml_2_cmake.py

```diff
@@ -73,6 +76,20 @@ def get_dependency_values(key, depends):
     return dependencies
 
 
+def escape_back_slash(line):
```

Comment:
Since `backslash` is a single word and the function should escape multiple this should be `escape_backslashs`.

But actually I was expecting the function actually escaped a `\` with a `\\`. Probably calling it something like `normalize_paths` or something similar would be clearer.